packages/access-api/migrations/0006_add_storage_provider_consumer_schema.sql

```diff
@@ -0,0 +1,21 @@
+-- Migration number: 0006 	 2023-03-02T16:40:04.407Z
+
+/*
+goal: add a table to keep track of the storage provider for a space.
+to enable https://github.com/web3-storage/w3protocol/issues/459
+*/
+
+CREATE TABLE
+    -- provision: the action of providing or supplying something for use
+    -- use case: representing the registration of a storage provider to a space
+    IF NOT EXISTS provisions (
+        id INTEGER NOT NULL PRIMARY KEY,
```

Comment:
Mind adding comment also what is the `id` here ? I'll probably discover as I read more of the pr, but comment here would be helpful

Reply:
It's meant to just be an auto incrementing id that other tables could join to.
I considered not even having it. wdyt about having it at all?

Reply:
added a comment. https://github.com/web3-storage/w3protocol/pull/462/commits/184c55a8dace6f56b92fbc7a99d10ca81975d666

fwiw it will not be autoincrementing it will be a 64-bit ROWID https://www.sqlite.org/lang_createtable.html#rowid

Reply:
https://github.com/web3-storage/w3protocol/pull/462/commits/40a4f1612d23c1d61d8bc68821f801d3c15587a1